rl_loop/train_and_validate.py

```diff
@@ -118,6 +121,8 @@ def validate_holdout_selfplay():
 
 def validate_pro():
     """Validate on professional data."""
+    if not FLAGS.pro_dataset:
+        print("*** --pro_dataset not set, skipping pro validation ***")
```

Comment:
add a return here?

Reply:
also, maybe put this print in loop() and gate it in loop() so that you don't endlessly print out this message.